docs/book/storage/adapter.md

```diff
@@ -782,6 +784,47 @@ Name | Data Type | Default Value | Description
 - Associative array: `['host' => <host>[, 'port' => <port>[, 'timeout' => <timeout>]]]`
 - List: `[<host>[, <port>, [, <timeout>]]]`
 
+## RedisCluster Adapter
+
+`Laminas\Cache\Storage\Adapter\RedisCluster` stores cache items over the redis cluster protocol
+using the PHP extension [redis](https://github.com/phpredis/phpredis).
```

Comment:
I suggest to add a link for the cluster support. (the link goes to the readme, not to any blob file in the repository)
```suggestion
`Laminas\Cache\Storage\Adapter\RedisCluster` stores cache items over the [Redis cluster](https://github.com/phpredis/phpredis#redis-cluster-support) protocol
using the PHP extension [PhpRedis](https://github.com/phpredis/phpredis).
```